### PR TITLE
#166272289 Chore Set up redux

### PR DESCRIPTION
**Pull request by despeauxz**
#### What does this PR do?
This PR sets up Redux into the application
#### Description of Task to be completed?
- Set up Redux to the application
#### How should this be manually tested?
- clone the repo
- install the packages via `npm install`
- start the dev-server with `npm run start:dev`
- install `Redux chrome devtool` as an extension on chrome
- Open your console and check for the `Redux` tab
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166272289](https://www.pivotaltracker.com/story/show/166272289)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: src/components/App/index.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+
+const App = () => (
```
Comment (fidelisojeah) [inline src/components/App/index.jsx]: I remember mentioning you guys should not create your components like this this morning

```suggestion
function App() {
  return (
```

**Review by fidelisojeah — CHANGES_REQUESTED** [review]

File: src/components/App/index.jsx (hunk 1 above)
Comment (despeauxz) [reply src/components/App/index.jsx]: I'll fix it right now, thanks

**Review by fidelisojeah — DISMISSED** [review]

**Review by adex001 — DISMISSED** [review]
Looks good to me.

**Review by adex001 — DISMISSED** [review]
Looks good. Works great! keep it up

**Review by EMacco — DISMISSED** [review]

**Review by EMacco — APPROVED** [review]
Good job

**Review by GerrardE — APPROVED** [review]

**Pull request merged**